### PR TITLE
Small changes

### PR DESCRIPTION
**Pull request by nperozzi**
These are some small changes that I have detected while reviewing the code after being two weeks away from it.

### CODE REVIEW

File: lib/atmega328/UART.cpp [hunk 1]
```diff
@@ -14,7 +14,7 @@ static uint8_t tx_queue_buffer[rx_queue_size];
```
Comment (emanuelen5) [inline lib/atmega328/UART.cpp:14]: The `tx_queue_buffer` should be `rx_queue_buffer` as well

**Comment by emanuelen5** [comment]
Other than that, it looks good

File: lib/atmega328/UART.cpp (hunk 1 above)
Comment (nperozzi) [reply lib/atmega328/UART.cpp:14]: Ups! sorry. I meat to change both but I must have forgotten one of them. It is fixed now.

**Review by emanuelen5 — APPROVED** [review]

**Comment by nperozzi** [comment]
I forgot to change the use of the rx buffer so it was not passing the tests. Now it is fixed and tests passed. I guess that you can merge this now.

**Pull request merged**